<ui version="4.0" >
 <class>AuthDialog</class>
 <widget class="QWidget" name="AuthDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>335</width>
    <height>193</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="MinimumExpanding" hsizetype="MinimumExpanding" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item rowspan="6" row="0" column="0" >
    <layout class="QVBoxLayout" name="verticalLayout" >
     <item>
      <widget class="QLabel" name="lblPixmap" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string notr="true">Lock Icon here</string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
       <property name="buddy" >
        <cstring>lePassword</cstring>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer" >
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>20</width>
         <height>92</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="1" colspan="3" >
    <layout class="QVBoxLayout" name="verticalLayout_2" >
     <item>
      <widget class="QLabel" name="lblHeader" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="MinimumExpanding" hsizetype="MinimumExpanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string notr="true">&lt;b>Header is here!&lt;/b></string>
       </property>
       <property name="wordWrap" >
        <bool>true</bool>
       </property>
       <property name="buddy" >
        <cstring>lePassword</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblContent" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="MinimumExpanding" hsizetype="MinimumExpanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string notr="true">&lt;i>Content&lt;/i></string>
       </property>
       <property name="wordWrap" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="1" colspan="3" >
    <widget class="KTitleWidget" name="errorMessageKTW" />
   </item>
   <item row="3" column="1" colspan="2" >
    <widget class="QLabel" name="lblPassword" >
     <property name="text" >
      <string>Password:</string>
     </property>
     <property name="buddy" >
      <cstring>lePassword</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="3" >
    <widget class="KLineEdit" name="lePassword" >
     <property name="echoMode" >
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="passwordMode" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="3" >
    <widget class="QCheckBox" name="cbRemember" >
     <property name="text" >
      <string>Remember authorization</string>
     </property>
     <property name="checked" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="2" colspan="2" >
    <widget class="QCheckBox" name="cbSessionOnly" >
     <property name="text" >
      <string>For this session only</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" >
    <spacer name="horizontalSpacer" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>15</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1" colspan="3" >
    <widget class="KComboBox" name="userCB" />
   </item>
  </layout>
  <zorder>errorMessageKTW</zorder>
  <zorder>lblPassword</zorder>
  <zorder>lePassword</zorder>
  <zorder>cbRemember</zorder>
  <zorder>cbSessionOnly</zorder>
  <zorder>horizontalSpacer</zorder>
  <zorder>userCB</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KTitleWidget</class>
   <extends>QWidget</extends>
   <header>ktitlewidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>userCB</tabstop>
  <tabstop>lePassword</tabstop>
  <tabstop>cbRemember</tabstop>
  <tabstop>cbSessionOnly</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cbRemember</sender>
   <signal>toggled(bool)</signal>
   <receiver>cbSessionOnly</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>259</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel" >
     <x>161</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
